libs/ui-libraries/src/lib/repo-info/repo-info.component.scss

```diff
@@ -0,0 +1,36 @@
+@import 'libs/theme/theme.scss';
+@import 'bootstrap-utilities';
```

Comment:
Aren't we importing this in the toolbar as well?
We REALLY need to globalize our styles.